__tests__/components/ui/infotooltip.test.tsx

```diff
@@ -0,0 +1,23 @@
+import { fireEvent, render, screen } from "@testing-library/react";
```

Comment:
This file is just the original `infoicon.test.tsx` file renamed to `infotooltip.test.tsx`